src/server/src/Server.cpp

```diff
@@ -180,7 +193,6 @@ std::string Server::set_position(const double x, const double y, const double z)
     return "OK";
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
and again